src/components/tables/ControllerStatusHistory/Rows.tsx

```diff
@@ -0,0 +1,50 @@
+import { TableCell, TableRow } from '@mui/material';
+import { orderBy } from 'lodash';
+import ControllerErrors from '../cells/entityStatus/ControllerErrors';
+import ControllerStatus from '../cells/entityStatus/ControllerStatus';
+import TimeStamp from '../cells/TimeStamp';
+import { RowProps, RowsProps } from './types';
+
+function Row({ row }: RowProps) {
+    return (
+        <TableRow>
+            {row.detail ? (
+                <ControllerStatus detail={row.detail} status={row.result} />
+            ) : (
+                <TableCell />
+            )}
+
+            {row.created ? (
+                <TimeStamp enableExact time={row.created} />
+            ) : (
+                <TableCell />
+            )}
+
+            {row.completed ? (
+                <TimeStamp enableExact time={row.completed} />
+            ) : (
+                <TableCell />
+            )}
```

Comment:
This is really interesting... our tables (and all the "helper" functions/components) really are focused on tables with data that is always there.